{"Functions": {"XPathFunctionException(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: The detail message."]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a new XPathFunctionException with the specified detail message. The cause is not initialized. If message is null, then a NullPointerException is thrown.", "history": "Added in API level 8", "FullName": "public XPathFunctionException (String message)"}, "XPathFunctionException(java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#Throwable_E@", "Throwable: The cause."]], "Throws": [["@B_java#lang#NullPointerException_E@", "if cause is null."]], "SeeAlso": [], "Permissions": [], "Description": "Constructs a new XPathFunctionException with the specified cause. If cause is null, then a NullPointerException is thrown.", "history": "Added in API level 8", "FullName": "public XPathFunctionException (Throwable cause)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_javax#xml#xpath#XPathException_E@", "@B_javax#xml#xpath#XPathExpressionException_E@"], "ClassName": "javax.xml.xpath.XPathFunctionException", "ClassDesc": "XPathFunctionException represents an error with an XPath function."}